### PR TITLE
Feature/tour 7 add tour page

### PR DESCRIPTION
**Pull request by markus-roe**
TOUR-7 Add Tour Page
TOUR-14 Edit Tour Page
TOUR-16 Tours Page

### CODE REVIEW

File: Client/Models/Tour.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-    public string Name { get; set; }
-    public string Description { get; set; }
-    public string TransportType { get; set; }
-    public string From { get; set; }
-    public string To { get; set; }
-    public int Popularity { get; set; }
+    public string Name { get; set; } = "";
+    public String From { get; set; } = "";
+    public String To { get; set; } = "";
+    public double EstimatedTime { get; set; } = 0;
```
Comment (mustermaxs) [inline Client/Models/Tour.cs:7]: TimeSpan?

File: Client/Models/Tour.cs [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
+    public string Description { get; set; } = "";
+    public string TransportType { get; set; } = "";
```
Comment (mustermaxs) [inline Client/Models/Tour.cs:9]: TransportType Enum verwenden?

File: Client/Models/Tour.cs [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
+    public int Difficulty { get; set; } = 0;
```
Comment (mustermaxs) [inline Client/Models/Tour.cs:10]: float

File: Client/Models/Tour.cs [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
+    public int Rating { get; set; } = 0;
```
Comment (mustermaxs) [inline Client/Models/Tour.cs:11]: float

**Review by mustermaxs — CHANGES_REQUESTED** [review]

File: Client/Models/Tour.cs (hunk 1 above)
Comment (markus-roe) [reply Client/Models/Tour.cs:7]: double needed for number input

**Review by mustermaxs — APPROVED** [review]

**Pull request merged**